code/modules/research/rdconsole.dm

```diff
@@ -145,6 +145,111 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 		user << "<span class='notice'>You you disable the security protocols.</span>"
 		return 1
 
+/obj/machinery/computer/rdconsole/proc/get_task()
+	var/task = null
+	switch(screen)
+		if(0.0)

```

Comment:
Don't use decimals. Use 10s instead.

So `0.0` becomes `10`, `0.1` -> `11` etc, `1.0` -> `20` etc. The reason is that integers are preferred over floating point numbers whenever possible, as they're less odd to handle.